src/view/dialogliste/DialogListe.tsx

```diff
@@ -15,8 +15,7 @@ export function DialogListe() {
     const dialogData = hasData(dialoger) ? dialoger.data : [];
     const { dialogId } = useParams();
 
-    const visningCls = dialogId ? classNames(styles.dialogOversikt, styles.dialogValgt) : styles.dialogOversikt;
-
+    const visningCls = dialogId ? classNames(styles.dialogListe, styles.dialogValgt) : styles.dialogListe;
```

Comment:
kan vi komme oss rundt dette?
Lurer på om vi kan bli kvitt behovet for dialogId her. Hadde gjort stilaset rundt mye lettere

Reply:
Dialogliste må nesten vite hvilken dialog som er valgt for å markere korrekt i lista. Men det at man bruker dialogId her for å finne ut om lista skal vises i det hele tatt er kanskje litt rart og kan se om det kan flyttes til et annet sted